activerecord/test/cases/adapters/abstract_mysql_adapter/connection_test.rb

```diff
@@ -200,8 +200,7 @@ def test_get_and_release_advisory_lock
     got_lock = @connection.get_advisory_lock(lock_name)
     assert got_lock, "get_advisory_lock should have returned true but it didn't"
 
-    assert_equal test_lock_free(lock_name), false,
-      "expected the test advisory lock to be held but it wasn't"
+    assert_equal(false, test_lock_free(lock_name), "expected the test advisory lock to be held but it wasn't")
```

Comment:
Let's keep the error message on a separate line like before:

```suggestion
    assert_equal false, test_lock_free(lock_name),
      "expected the test advisory lock to be held but it wasn't"
```

Reply:
> Are these being parentheses from Rubocop's autocorrect? If so, that is unfortunate.

Ah, this should ideally be handled by RuboCop Minitest, but it seems to produce unexpected auto-correction code. I will fix this autocorrection logic of RuboCop Minitest.